x-pack/legacy/plugins/lens/public/editor_frame_plugin/mocks.tsx

```diff
@@ -85,11 +85,15 @@ export function createMockFramePublicAPI(): FrameMock {
 type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 
 export type MockedSetupDependencies = Omit<EditorFrameSetupPlugins, 'data'> & {
-  data: Omit<DataSetup, 'expressions'> & { expressions: jest.Mocked<DataSetup['expressions']> };
+  data: Omit<DataSetup, 'expressions'> & {
```

Comment:
This was in place to mock the expressions part of the data plugin, but as it's a separate plugin now you should be able to do this:
```ts
export type MockedSetupDependencies = Omit<EditorFrameSetupPlugins, 'expressions'> & {
   expressions: jest.Mocked<ExpressionsSetup>;
};
```